src/components/RestaurantList.js

```diff
@@ -0,0 +1,26 @@
+customElements.define(
+  'restaurant-list',
+  class RestaurantList extends HTMLElement {
+    constructor() {
+      super();
+    }
+
+    render(restaurants) {
+      this.innerHTML = /* html */ `
```

Comment:
이 주석은 왜 있는거에요?

Reply:
이어지는 코드가 HTML 마크업임을 알려주기 위해 작성했습니다. 또한 HTML Syntax를 하이라이팅해주어 휴먼 에러를 줄여주는 VSC Extension(`es6-string-html`)의 도움을 받기 위해 사용했습니다.
![image](https://user-images.githubusercontent.com/19235163/223336417-3edfd54f-bc54-4ac1-866a-7b655cdcfc93.png)

Reply:
> HTML Syntax를 하이라이팅해주어 휴먼 에러를 줄여주는 VSC Extension(es6-string-html)

좋네용 :) 

```suggestion
      /* html */ 
      this.innerHTML = `
```

저였다면 이렇게 썼을 것 같기는 하지만, 좋은 의도 알려주셔서 감ㅅㅏ합니다~